### ISSUE TITLE
Unable to send Ethereum from Wallet

### ISSUE THREAD

**Issue by JPSD** [ethereum/mist#3930]
Version: Binary
OS & Version: windows 8.1
Node version: `geth 1.8.8`
Number of blocks synchronized: 5705159

Hi , I am having problems sending Ethereum from my wallet.
This is the first time I have used this method to gain help.
I am no computer expert in fact probably a noob by your standards.
My  Ethereum Geth wallet is running the  latest version and is fully synched
I have tried upping the transaction fee and updating the time as suggested in other links.
The transaction show 0 of 12 then transaction fails in a yellow  communication box. 
I have tried 5 times now using different addresses and I cannot send Ethereum.

Is there a quick fix? 
 This seems a common problem but I see no real answers in other social media. If there is please direct me to them.

Thanking you in advance

JP

**Comment by JPSD** [comment]
In relation to my earlier request I Have found this on the console but have no idea what it means

Uncaught Error: No callback given, sync calls are not possible anymore in Mist. Please use only async calls. at EthereumProvider._addResponseCallback (<anonymous>:1293:13) at EthereumProvider.send (<anonymous>:1337:10) at s.send (2a3fe1c….js:187) at r.e [as blockNumber] (2a3fe1c….js:187) at <anonymous>:1:9

Is this any help?

**Comment by JPSD** [comment]
Hi, 
Further to the above request for help.
All the withdrawals I tried to send but was told had been unsuccessful all went through 6 hours ago. 
Having been told the transaction had  failed I tried several times to send the same amount. Now I find they have ALL been sent even though I was told they had failed. There is definitely a problem with the wallet. I know they have been sent as I owned the addresses they were sent to, however my wallet shows the new subtotal but does not show  withdrawals. Ether scan does show the withdrawals.
Any answers for me?

JP
